### ISSUE TITLE
Broken fonts after 0.7.2 update

### ISSUE THREAD

**Issue by CodedNil** [amPerl/egui-phosphor#27]
After updating from 0.7.1 to 0.7.2 all my fonts have broken and lowercase letters don't display:
![image](https://github.com/user-attachments/assets/8cddf137-34a8-4a8b-9d63-0a5477793335)

My set up is very simple and all I'm doing is as per the examples
```rust
        let mut fonts = egui::FontDefinitions::default();
        egui_phosphor::add_to_fonts(&mut fonts, egui_phosphor::Variant::Regular);
        cc.egui_ctx.set_fonts(fonts);
```

**Comment by amPerl** [comment]
Very strange. Your example looks fine, but I'm unable to reproduce this. The only change was pushing the phosphor font to the front of the list rather than the back. Could you try cloning this crate and running `cargo run --example list_all` to see if it's some sort of system-wide problem?

**Comment by bircni** [comment]
Fixed this in #28 and you can see an example of this happening in #29

**Comment by amPerl** [comment]
Ah, that makes sense, the list example simply did not have lowercase characters. What a strange issue, I guess there's something in the font that makes it falsely define glyphs in ranges outside of its own icons. I'll revert it for now.

**Comment by bircni** [comment]
@amPerl maybe also yank the 0.7.2

**Comment by bircni** [comment]
can be closed...

**Comment by amPerl** [comment]
Pulled 0.7.2 and published 0.7.3, @CodedNil can you verify this works for you?

**Comment by CodedNil** [comment]
Works great, thanks for the fast fix
